src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -137,8 +154,10 @@
      * @param { boolean } atBookCenter - optional
      */
     var toggleRouter = function toggleRouter (br, e, atBookCenter) {
-      var isValidClickArea = atBookCenter ? isCenterClick(e) : isBackground(e.target);
-
+      var book = isBRcontainerScrollable() ? br.refs.$brContainer[0] : event.currentTarget;
+      var is1UpMode = br.constMode1up === br.mode;
+      var validBookClick = is1UpMode ? true : isCenterClick(e, book);
```

Comment:
Since `is1UpMode` will be a boolean, you can just use `is1UpMode || isCenterClick(e, book);`